### PR TITLE
Release Croatian translations

### PR DESCRIPTION
**Pull request by milotype**

### CODE REVIEW

File: templates/hr/access-default.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Ako moje osobne podatke prenosite u treću zemlju ili međunarodnu organizaciju, zahtijevam da me obavijestite o odgovarajućim zaštitnim mjerama u skladu s člankom 46. GDPR-a u vezi s prijenosom.
+[data_portability>
+Molim vas da mi omogućite pristup osobnim podacima u strukturiranom, uobičajenom i strojno čitljivom formatu, kako je utvrđeno u članku 20. stavka 1. GDPR-a.
+]
+Moj zahtjev izričito uključuje [runs>sljedeće kao i ]sve druge usluge i poduzeća za koje ste voditelj obrade, kako je definirano u članku 4. stavka 7. GDPR-a [runs>: {runs_list}].
+
+Kao što je utvrđeno u članku 12. stavka 3. GDPR-a, tražene podatke morate mi dostaviti bez odgađanja, a u svakom slučaju u roku od mjesec dana po primitku zahtjeva. Na osnovi članka 15. stavka 3. GDPR-a, na ovaj zahtjev morate odgovoriti bez naknade.
+[has_fields>
+Moji identifikacijski podaci:
+{id_data}
```
Comment (baltpeter) [inline templates/hr/access-default.txt:25]:
```suggestion
{id_data}]
```

File: templates/hr/access-tracking.txt [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Molim vas da mi omogućite pristup osobnim podacima u strukturiranom, uobičajenom i strojno čitljivom formatu, kako je utvrđeno u članku 20. stavka 1. GDPR-a.
+]
+Moj zahtjev izričito uključuje [runs>sljedeće kao i ]sve druge usluge i poduzeća za koje ste voditelj obrade, kako je definirano u članku 4. stavka 7. GDPR-a [runs>: {runs_list}].
+
+Kao što je utvrđeno u članku 12. stavka 3. GDPR-a, tražene podatke morate mi dostaviti bez odgađanja, a u svakom slučaju u roku od mjesec dana po primitku zahtjeva. Na osnovi članka 15. stavka 3. GDPR-a, na ovaj zahtjev morate odgovoriti bez naknade.
+
+Molim vas da me obavijestite o potrebnim identifikacijskim podacima (ID kolačića, ID uređaja oglašavanja, itd.) te da mi objasnite kako doći do tih podataka.
+[has_fields>
+Moji identifikacijski podaci:
+{id_data}
```
Comment (baltpeter) [inline templates/hr/access-tracking.txt:27]:
```suggestion
{id_data}]
```

**Review by baltpeter — APPROVED** [review]
Wow, that was quick. Thank you very much! :)

**Comment by milotype** [comment]
@baltpeter Thank you for the corrections!

**Pull request merged**